zeus/monitor/energy.py

```diff
@@ -188,12 +188,13 @@ def __init__(
         except ZeusCPUInitError:
             self.cpus = EmptyCPUs()
         except ZeusRAPLPermissionError as err:
-            raise RuntimeError(
-                "SYS_ADMIN capability is required to modify GPU power limits. See "
-                "https://ml.energy/zeus/getting_started/#system-privileges "
-                "for more information or disable CPU measurement by passing cpu_indices=None to "
-                "ZeusMonitor"
-            ) from err
+            if cpu_indices:
+                raise RuntimeError(
+                    "SYS_ADMIN capability is required to read RAPL files. See "
```

Comment:
Is it `SYS_ADMIN` or `root`? They're not exactly same things. The former is one thing under the Linux security capability.

Reply:
It seems like it is `root` according to this code carbon [thread](https://github.com/mlco2/codecarbon/issues/244#issuecomment-1107754478).

Reply:
Yeah, I suppose the file's owner is `root` and other users are not given read access, so it has to be exactly `root`.